tasks/main.yml

```diff
@@ -12,7 +12,7 @@
 
 - name: Install Dell DSU repo
   environment: "{{ proxy_env|default({}) }}"
-  shell: "wget -q -O - http://linux.dell.com/repo/hardware/dsu/bootstrap.cgi | bash"
+  shell: "curl -O https://linux.dell.com/repo/hardware/dsu/bootstrap.cgi && bash bootstrap.cgi"
```

Comment:
maybe even

` curl -O https://linux.dell.com/repo/hardware/dsu/bootstrap.cgi && sed -e s/IMPORT_GPG_CONFIRMATION=\"na\"/IMPORT_GPG_CONFIRMATION=\"no\"/g bootstrap.cgi |bash`  since one might not want to install gpg keys..

Reply:
1) http->https might be a breaking change for some legacy cases. But, I suppose nowadays it would be fair to presume everyone using this role should be able to verify the site up to the signing CA. Should be fine but please make a note of this before merging.

2) The "maybe even" proposal would be in line with @Klaas- 's comment in https://github.com/CSCfi/ansible-role-dell-firmware-upgrade/issues/19#issuecomment-502968606 , and the current implementation of doing the rollout. One minor detail: Is there any particular reason to save to a file (instead of just `curl|sed|bash`)?

Reply:
1. we can't forever bring everyones legacy cases along. http->https is more or less standard nowadays.
2. I would be very pleased about "maybe even" approach, but in order to simplify the changes, I tried to bring proposal as minimal as possible, in order to proceed. 
2.minor. Saving a file was idea just for changeing na => no. There are more elegant ways to do it. That was just a supersimple and fast sed hack.